### PR TITLE
Craftable shoffy cells - moleminer gauntlet buff - smithing buffs

### PR DESCRIPTION
**Pull request by FarmWizard**
## About The Pull Request
<1/2 normal capacity
these CAN be used for enhanced cells
these are tied to weaponsmith
these are not added to skillbook

Also does the first pass of smithing rework
most weapons got a buff to damage
pickaxes got a buff
will need testing for balance

buff to moleminer gauntlet
>

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
add: Added 1/2 capacity craftable cells to weaponsmith
tweak: made the moleminer gaunlet better in general
balance: made smithing a bit more competitive
/:cl:

### CODE REVIEW

**Review by Superlagg — CHANGES_REQUESTED** [review]
Good PR, dunno why beauty did that. Just need some changes!

File: code/modules/smithing/smithed_items.dm [hunk 1]
```diff
@@ -101,23 +101,23 @@
 	var/qualname
 	switch(quality)
 		if(-1000 to -5)
-			qualname = "awful"
+			qualname = "heinous"
```
[the remark this answers is not in the document]
Comment (FarmWizard) [reply code/modules/smithing/smithed_items.dm:104]: Alright, it was like this before I touched it so I'll resort it.

File: code/modules/power/cell.dm [hunk 2]
```diff
@@ -588,6 +615,25 @@
...
+	name = "counterfeit electron charge pack"
+	desc = "An electron charge pack, typically used as ammunition for rapidly-firing energy weapons. This one looks slightly off, somehow."
+	icon = 'icons/fallout/objects/powercells.dmi' //TODO: Give a shitty icon
+	icon_state = "ecp-full"
+	maxcharge = 1200
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/stock_parts/cell/ammo/ecp/update_icon()
+	switch(charge)
+		if (601 to 1200)
```
Comment (GremlingSS) [inline code/modules/power/cell.dm:629]: `		if (601 to 1200)`
to:
`		if (601 to maxcharge)`

File: code/modules/power/cell.dm [hunk 3]
```diff
@@ -501,6 +501,25 @@
...
+	name = "shoddy microfusion cell"
+	desc = "A microfusion cell, typically used as ammunition for large energy weapons. This one looks a little dubious though."
+	icon = 'icons/fallout/objects/powercells.dmi' //TODO: give these bad icons
+	icon_state = "mfc-full"
+	maxcharge = 1000
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/stock_parts/cell/ammo/mfc/update_icon()
+	switch(charge)
+		if (501 to 1000)
```
Comment (GremlingSS) [inline code/modules/power/cell.dm:515]: `		if (501 to 1000)`
to
`		if (501 to maxcharge)`

**Review by GremlingSS — CHANGES_REQUESTED** [review]
ye

File: code/modules/power/cell.dm (hunk 3 above)
Comment (FarmWizard) [reply code/modules/power/cell.dm]: Poked it and yeah it's not updating the children's icons like it should. I'd say just deal with the bad cells flashing they're half-full!

Comment (FarmWizard) [reply code/modules/power/cell.dm]: Alright, changed everything to a math based system and now none of the cells are updating properly.

**Pull request closed without merging**